lightningd/plugin.c

```diff
@@ -813,6 +813,12 @@ static void plugin_manifest_cb(const char *buffer,
 		plugin_kill(
 		    plugin,
 		    "Failed to register options, methods, hooks, or subscriptions.");
+
+	/* If all plugins have replied to getmanifest and this is not
+	 * the startup init, configure them */
+	if (!plugin->plugins->startup && plugin->plugins->pending_manifests == 0)
+		plugins_config(plugin->plugins);
```

Comment:
Doesn't this result in us re-configuring all plugins every time we add a new plugin?

Reply:
No, 2ba5145a3e460684e5f65e5a09523238cf65dee5 introduces 
```C
list_for_each(&plugins->plugins, p, list) {
	if (p->configured)
		continue;
```